wavefront-istio-adapter/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+name: wavefront-istio-adapter
+version: 0.1.3
+description: A Helm chart for Wavefront by VMware Adapter for Istio
+home: https://github.com/vmware/wavefront-adapter-for-istio/
+sources:
```

Comment:
Also add a keywords section similar to https://github.com/wavefrontHQ/helm/blob/df3eaa72ba97ad9b870dbf4396e2b8d63eb5993d/wavefront-hpa-adapter/Chart.yaml#L6